widget/playerctl.lua

```diff
@@ -30,8 +31,12 @@ local function factory(args)
     local followtag     = args.followtag or false
     local settings      = args.settings or function() end
 
+    local default_icon_cache = gears.filesystem.get_cache_dir()
+    local default_icon_file = string.format("%s%s", default_icon_cache, 'playctrl_icon.jpg')
+
     local metadata = string.format("playerctl metadata")
     local status = string.format("playerctl status")
+    local icon_cmd = string.format("curl -o %s ", default_icon_file)
```

Comment:
New to lua, probably a built in lib I am missing that can do this better then curl.